inference-engine/tests/ie_test_utils/functional_test_utils/src/layer_test_utils/summary.cpp

```diff
@@ -133,21 +134,30 @@ void Summary::updateOPsStats(const std::shared_ptr<ngraph::Function> &function,
             ngraph::is_type<ngraph::op::Result>(op)) && isFunctionalGraph) {
             continue;
         } else if (ngraph::is_type<ngraph::op::TensorIterator>(op)) {
-            updateOPsStats(op->get_type_info(), status);
+            updateOPsStats(op->get_type_info(), status, className);
             auto ti = ngraph::as_type_ptr<ngraph::op::TensorIterator>(op);
             auto ti_body = ti->get_function();
-            updateOPsStats(ti_body, status);
+            updateOPsStats(ti_body, status, className);
         } else if (ngraph::is_type<ngraph::op::v5::Loop>(op)) {
-            updateOPsStats(op->get_type_info(), status);
+            updateOPsStats(op->get_type_info(), status, className);
             auto loop = ngraph::as_type_ptr<ngraph::op::v5::Loop>(op);
             auto loop_body = loop->get_function();
-            updateOPsStats(loop_body, status);
+            updateOPsStats(loop_body, status, className);
         } else {
-            updateOPsStats(op->get_type_info(), status);
+            updateOPsStats(op->get_type_info(), status, className);
         }
     }
 }
 
+void Summary::savePartReport(const char* className, const char* opName, unsigned long passed, unsigned long failed,
```

Comment:
remove debug code (or move under the macros)

Reply:
Moved under the macros